README.md

```diff
@@ -1 +1 @@
-# cluckr
+# Allauda
```

Comment:
Shall we change it to one 'l' now that I updated the logo?

Reply:
I'll update the README after this PR ;)